client/components/domains/register-domain-step/index.jsx

```diff
@@ -498,6 +501,43 @@ class RegisterDomainStep extends Component {
 		);
 	}
 
+	renderQuickFilters() {
+		// TODO: Confirm where should we load these tlds from
+		const items = [
+			{ key: 'all', text: 'All' },
+			{ key: 'blog', text: '.blog' },
+			{ key: 'com', text: '.com' },
+			{ key: 'news', text: '.news' },
+			{ key: 'email', text: '.email' },
+			{ key: 'info', text: '.info' },
+			{ key: 'help', text: '.help' },
+			{ key: 'press', text: '.press' },
+			{ key: 'report', text: '.report' },
+		];
```

Comment:
I think it's okay to hardcode this for the experiment, but once we conclude that we need this.
Might be better to load from the back end. Any thoughts?